backend/src/ml_space_lambda/initial_app_config/lambda_function.py

```diff
@@ -44,5 +46,24 @@ def lambda_handler(event, context):
     ]
 
     app_configuration_dao.update(config)
+    update_dynamic_roles_with_notebook_policies()
 
     generate_html_response(200, "Successfully updated app config")
+
+
+def update_dynamic_roles_with_notebook_policies():
+    env_vars = get_environment_variables()
+
+    if env_vars["MANAGE_IAM_ROLES"] == "false":
```

Comment:
I think you just want `if not env_vars["MANAGE_IAM_ROLES"]:` here. I think that strings are technically truthy. I think technically the value some users may use is ""

Everywhere else in the codebase we do `if env_variables["MANAGE_IAM_ROLES"]:`

Reply:
Looks like for consistence I shouldn't be setting true/false in the environment variables but "true"/"". I updated the lambda definition so this usage matches the rest of the project.